extract.py

```diff
@@ -120,6 +130,28 @@ def extract_database(args):
     return output_rows
 
 
+def extract_csv(args, mapping=None, csvfile="", init_id=100000, date_format="%y%m%d"):
+    if mapping is None:
+        mapping = dict()
+    mapping["date_format"] = date_format
+    output_rows = []
+    report = get_report()
+    person_id = init_id
+    with open(csvfile, "r") as datasource:
+        rows = csv.DictReader(datasource)
+        for row in rows:
+            handled_row = handle_row(row, report, mapping)
```

Comment:
One thing that this feature doesn't do which, as I look at the hfc translation code I realize that I might want to add is  specific value handling for gender. @dehall, should I do that?

Reply:
`handle_row()` already does this by means of [`utils.data_readercase_insensitive_lookup()`](https://github.com/mitre/data-owner-tools/blob/csv-translation/utils/data_reader.py/#L93), which I also expanded a little so as to handle default values if the lookup is unsuccessful

Reply:
I may be missing something the code is doing, but I meant lookup along the lines of "if the value from the source columns is `Female` use `F` in the output column, if the value is `Male` use `M` "

Reply:
Ahhh. I was confused earlier about what you were talking about, but I get it now. Do we want to hard-code those regularization rules into the script, or pull them from the config file?

Reply:
I think if we hardcode anything we'll be surprised by what we eventually find in the real world. (Maybe somebody uses 0=unknown, 1=female, 2=male in their system)
If it can come from the config that would be ideal